src/features/rewards/tab/index.tsx

```diff
@@ -0,0 +1,67 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import {
+  RewardsTabWrapper,
+  StyledSlider,
+  StyledBullet,
+  StyledText,
+  StyledSwitch
+} from './style'
+
+export interface Props {
+  id?: string
+  tabOneChecked?: boolean
+  tabTitles?: string[]
+  testId?: string
+  tabIndexSelected?: number
+  onChange?: (event: React.MouseEvent<HTMLDivElement>) => void
+}
+
+export default class Tab extends React.PureComponent<Props, {}> {
+  static defaultProps = {
+    tabIndexSelected: 0
+  }
+
+  getTabs = (tabTitles: string[]) => {
+    const tabs: React.ReactNode[] = tabTitles.map((title: string, i: number) => {
+      return (
+        <StyledText
+          left={i === 0}
+          key={`tab-${i}`}
+          selected={i === this.props.tabIndexSelected}
+        >
+          {title}
+        </StyledText>
+      )
+    })
+    return tabs
+  }
+
+  render () {
+    const {
+      id,
+      testId,
+      tabTitles,
+      onChange,
+      tabIndexSelected
+    } = this.props
+
+    if (!tabTitles || tabTitles.length !== 2) {
+      console.warn('Rewards Tab currently supports 2 tab titles')
+      return null
+    }
+
+    return (
+      <RewardsTabWrapper id={id}>
```

Comment:
Do we need this extra wrapper element? Doesn't seem to do much.

Reply:
We should be able to support all of the above mentioned @pete. Let me know why it can’t.

Reply:
We should be able to support all of the above mentioned @pete. Let me know why it can’t.

Reply:
Fyi, I don’t want to speak for @ryanml but he may not have taken those into account because he was implementing it for the android vs.

Reply:
@jenn-rhim what are the expected active and hover states for this component?